### ISSUE TITLE
Release plan?

### ISSUE THREAD

**Issue by thenesk** [avast/retdec#1062]
I am impressed with this projects capabilities and would love a new release.  I see some issues have been fixed since the last release and there have been a variety of commits... is it almost time for another release?

**Comment by Brubbish** [comment]
Yeah, and I believe that the README and WIKI can also be updated, the current README and WIKI are toooo out of date

**Comment by s3rvac** [comment]
@PeterMatula Could you please describe the current plan here?
